### PR TITLE
Please bump monotonic to 1.3

### PR DESCRIPTION
**Pull request by sodre**
This recipe update is needed by PR conda-forge/staged-recipes#2927

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -29,10 +29,12 @@ test:
 
 about:
   home: https://github.com/atdt/monotonic
-  license: Apache License Version 2.0
+  license: Apache
```
Comment (jakirkham) [inline recipe/meta.yaml]: Would be good to keep the `2.0`.

**Comment by jakirkham** [comment]
Also could use a re-render.

**Comment by sodre** [comment]
@jakirkham 
sorry about the license issue.

**Comment by jakirkham** [comment]
np @sodre. Thanks for correcting it.

**Comment by sodre** [comment]
it has been 8 days since I opened this PR, it is a blocker for conda-forge/staged-recipes#2927. @isuruf, are you able to merge it yourself?

cc: @conda-forge/monotonic

**Comment by isuruf** [comment]
Nope, I don't have permission.

**Pull request merged**